src/ButterCMS/ButterCMS.php

```diff
@@ -24,18 +27,23 @@ public function __construct($authToken)
     {
         $this->authToken = $authToken;
 
-        $this->client = new Client();
+        $this->client = new Client([
+            'headers' => [
+                'X-Butter-Client' => 'PHP/2.3.0',
```

Comment:
Can you make the `2.3.0` driven off of a variable/CONST in this package?  Our other SDK's manage version number via a CONSTANT.